src/app/components/graphics/graphics.component.ts

```diff
@@ -0,0 +1,83 @@
+import {Component, ElementRef, Input, OnInit, ViewChild} from '@angular/core';
+import Chart from 'chart.js';
+import {GraphicsData} from './graphics-data';
+
+@Component({
+  selector: 'app-graphics',
+  templateUrl: './graphics.component.html',
+  styleUrls: ['./graphics.component.css']
+})
+export class GraphicsComponent implements OnInit {
+  @ViewChild('canvas', {static: true})
+  canvas: ElementRef<HTMLCanvasElement>;
+  @Input() dataset: number[][];
+  datasets: GraphicsData[] = [];
+  chart = Chart;
```

Comment:
why do you created this variable?

Reply:
It is required for the library

Reply:
I believe you should type it here and set it later